typings/@chainlink/index.d.ts

```diff
@@ -19,13 +20,34 @@ declare module '@chainlink/types' {
   export type Callback = (statusCode: number, data?: any) => void
   export type AdapterHealthCheck = (callback: Callback) => any
 
+  import { AxiosRequestConfig } from 'axios'
+  export type Config = {
+    apiKey?: string
+    network?: string
+    returnRejectedPromiseOnError?: Boolean
+    verbose?: boolean
+    api: Partial<AxiosRequestConfig>
+  }
+
+  /* RESPONSES */
+  export type DataResponse<R, P> = {
+    result: R
+    payload?: P
+  }
+
+  export type SequenceResponseData<R> = {
```

Comment:
If the one above is named `DataResponse` let's also rename this as `SequenceDataResponse `

Reply:
We'll also need to improve and structure these types in the future, but this should be good for now.

Reply:
Agreed, will open an issue for this so we remember.